Documentation/api.md

```
@@ -411,6 +413,21 @@ The `NodeExporterCollectorTcpStatConfig` resource works as an on/off switch for
 
 [Back to TOC](#table-of-contents)
 
+## NodeExporterCollectorZoneinfoConfig
+
+#### Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)